contracts/test/TokenDistributionMock.sol

```diff
@@ -17,7 +18,7 @@ contract TokenDistributionMock is ITokenDistribution {
         over = false;
     }
 
-    function finalize() external {
```

Comment:
A previous change allowed this function to be called by anyone at anytime. If this function is called before all the grants are added in the genesis migration script then the grants cannot be added afterwords. Adding the `onlyOwner` modifier ensures that the deployment account can only call `finalize` and allows the deployment account to make sure to add all the grants before calling this function